core/src/impl/Kokkos_ViewTracker.hpp

```diff
@@ -77,6 +81,44 @@ struct ViewTracker {
   ViewTracker(const ViewTracker& vt) noexcept
       : m_tracker(vt.m_tracker, view_traits::is_managed) {}
 
+  KOKKOS_INLINE_FUNCTION
+  explicit ViewTracker(ParentView& dst, const ParentView& src) noexcept
+      : m_tracker() {
+#if defined(KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_HOST)
+    if (view_traits::is_managed &&
+        Kokkos::Impl::SharedAllocationRecord<void, void>::tracking_enabled()) {
+      m_tracker.assign_direct(src.m_track.m_tracker);
+#if defined(KOKKOS_ENABLE_VIEW_HOOKS)
+      if (Kokkos::Experimental::ViewHooks::is_set())
+        Kokkos::Experimental::ViewHooks::call(dst, src);
```

Comment:
Is there a more descriptive name than `call`? What is this function actually supposed to do?

Reply:
ViewHooks::call is the top level function that calls the user specified callback functor (to process the views)

I am open to suggestions on the name/s

Reply:
I don't know that this is better, but in subscription stuff I've seen `notify` for `invoke/call`, so `notify_copy` might be good?